api/lib/application/organizations/organization-controller.js

```diff
@@ -84,6 +85,23 @@ module.exports = {
     return membershipSerializer.serialize(memberships, pagination);
   },
 
+  async getMultiplePDFAttestations(request, h) {
```

Comment:
J'ai souvenir d'avoir fait la remarque à l'époque quand c'était encore en doing, je pense qu'il faut bien mettre l'accent sur le fait que c'est une feature aujourd'hui dispo uniquement pour le SCO is managing students.
Donc j'ajouterai bien cette mention dans le descriptif de la route dans le index.js, mais aussi davantage au niveau du nommage.
Ici dans le handler de controller, ça gagne à être plus précis le fait que c'est pas seulement getMultiplePDFAttestations (par ailleurs confus car on dirait qu'on va avoir plusieurs fichiers pdf...), mais bien les attestations d'une classe entière. Je me débarasserai de ce terme multiple au maximum, car c'est pas multiple, c'est toutes celles d'une classe.
`downloadCertificationAttestationsForDivision` ? (même la notion de pdf me paraît pas utile car une attestation chez nous est toujours sous le format pdf)